app/common/lib/bookmarkFoldersUtil.js

```diff
@@ -26,7 +27,7 @@ const getNextFolderIdItem = (folders) => {
 }
 
 const getNextFolderId = (folders) => {
-  const defaultFolderId = 0
+  const defaultFolderId = 1
```

Comment:
why does the default folder Id have to change here?

Reply:
because it was wrong. Default folderId can't be 0, because 0 is bookmark toolbar and -1 is other bookmarks

Reply:
I noticed this when I was adding `buildFolder` function and adding unit tests for it

Reply:
and if don't have any folders that means that it should be 1 and not 0

Reply:
this was not a problem before (and it still isn't) because we are passing at least empty Map into this function and that's why this if is not triggered. What we could do is to check if map is  empty and then trigger this if. But in unit test first thing that you do is pass null into it :smile:

Reply:
ok thanks.